packages/core/src/components/Tag/Tag.stories.tsx

```diff
@@ -243,3 +245,176 @@ export const TagArray: StoryObj<HvTagProps> = {
     );
   },
 };
+
+export const Selectable: StoryObj<HvTagProps> = {
```

Comment:
Docs-wise, the `Selectable` & `SelectableControlled` samples show pretty much the same code, as the functions are rendered 😞